utils/utils.go

```diff
@@ -263,6 +264,15 @@ func CreateClient(conf NetConf) (*client.Client, error) {
 	return calicoClient, nil
 }
 
+func IsReady(client *client.Client) bool {
+	kvPair, err := client.Backend.Get(model.ReadyFlagKey{})
+	if err != nil {
+		return true
```

Comment:
Maybe we should log a line here maybe a warning level saying can't get to the datastore or something. WDYT?

Reply:
Yeah, there are sort of three cases here:
- datastore is reachable, not ready
- datastore is reachable, ready
- datastore is not reachabe

We want to make sure the correct logging and errors returned for each. We might want to change the signature of this function to return an error, so we can properly bubble up the connection error.